tests/test_unit/test_sample_data.py

```diff
@@ -1,42 +1,49 @@
 """Test suite for the sample_data module."""
 
+from pathlib import Path
 from unittest.mock import MagicMock, patch
 
 import pooch
 import pytest
 from requests.exceptions import RequestException
 from xarray import Dataset
 
-from movement.sample_data import _fetch_metadata, fetch_dataset, list_datasets
+from movement.sample_data import (
+    _fetch_metadata,
+    fetch_dataset,
+    fetch_dataset_paths,
+    list_datasets,
+)
+
+# Define sample datasets for parametrization
+SAMPLE_DATASETS = {
+    "SLEAP_three-mice_Aeon_proofread.analysis.h5": {
+        "fps": 50,
+        "frame_file": "three-mice_Aeon_frame-5sec.png",
+        "video_file": "three-mice_Aeon_video.avi",
+    },
+    "DLC_single-wasp.predictions.h5": {
+        "fps": 40,
+        "frame_file": "single-wasp_frame-10sec.png",
+        "video_file": None,
+    },
+    "LP_mouse-face_AIND.predictions.csv": {
+        "fps": 60,
+        "frame_file": None,
+        "video_file": None,
+    },
+    "VIA_multiple-crabs_5-frames_labels.csv": {
+        "fps": None,
+        "frame_file": None,
+        "video_file": None,
+    },
```

Comment:
We should add TRex here as well
```suggestion
    },
    "TRex_five-locusts.zip": {
        "fps": 5,
        "frame_file": "locusts-noqr_n-5_date-20250117_frame-bg.png",
        "video_file": None,
    },
```